siobrultech_protocols/gem/api.py

```diff
@@ -81,6 +91,26 @@ async def get_serial_number(protocol: BidirectionalProtocol) -> int:
     formatter=lambda dt: f"{CMD_SET_DATE_AND_TIME}{dt.strftime('%y,%m,%d,%H,%M,%S')}",
     parser=lambda response: response == "DTM",
 )
+SET_HTTP_POST_DOMAIN = ApiCall[str, bool](
+    formatter=lambda url: f"{CMD_SET_HTTP_POST_DOMAIN}{url}",
```

Comment:
Docs say you need a CR at the end of this one and the next one.

Though note that this is *not* setting the endpoint to which the GEM is sending binary packets. This is the HTTP POST option. (So if you're doing this for me, don't bother ;D)

Reply:
> Docs say you need a CR at the end of this one and the next one.

Oh, good catch.  For some reason, I thought all of these needed a CR at the end of them, but looks like I was wrong.

> Though note that this is not setting the endpoint to which the GEM is sending binary packets. This is the HTTP POST option. (So if you're doing this for me, don't bother ;D)

Clearly I misunderstood what you needed.  Have you posted in their forum asking about this API?  I'm sure it exists since they can set it from their client, I think.  Might just need updated docs.

Reply:
I haven't, but they've been responsive in the past so that's a good idea and I just did.

Reply:
I'm just going to remove these for now.

Reply:
Looking at your post and the screenshot you provided, it occurs to me that that might be something specific to the wifi/ethernet module, which I don't even have (I have just the ethernet module).  It might be a configuration of that specific device, as opposed to the GEM itself (which we can certainly figure out how to support).